zds/tutorialv2/models/models_database.py

```diff
@@ -135,8 +141,12 @@ def textual_type(self):
         """
         if self.is_article():
             return _(u"L'Article")
-        else:
+        elif self.is_opinion():
+            return _(u"Le Billet")
+        elif self.is_tutorial():
             return _(u"Le Tutoriel")
+        else:
+            return _(u"Le Contenu")
```

Comment:
programmation défensive.


Le 12/05/2016 à 20:55, Gérard Paligot a écrit :
>
> In zds/tutorialv2/models/models_database.py 
> <https://github.com/zestedesavoir/zds-site/pull/3606#discussion_r63076749>:
>
> >              return _(u"Le Tutoriel")
> > +        else:
> > +            return _(u"Le Contenu")
>
> Il y a des cas où on rentre ici ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/zestedesavoir/zds-site/pull/3606/files/a9d32d475ae67dda96b6bb54ce9174fad3ec1595#r63076749>
>